includes/MediaWiki/Request/EditRequest.php

```diff
@@ -17,23 +17,29 @@ class EditRequest {
 	/** @var Item */
 	private $entity;
 
+	/** @var string */
+	private $token;
+
 	/** @var ReconciliationItem[] */
 	private $otherItems;
 
 	/**
 	 * @param PropertyId $reconcilePropertyId Must have the datatype "url".
 	 * @param Item $entity
+	 * @param string $token
 	 * @param ReconciliationItem[] $otherItems Other items that need to be created,
 	 * since their URLs were referenced in $entity but they did not exist yet.
 	 */
 	public function __construct(
 		PropertyId $reconcilePropertyId,
 		Item $entity,
+		string $token,
 		array $otherItems = []
 	) {
 		$this->reconcilePropertyId = $reconcilePropertyId;
 		$this->entity = $entity;
 		$this->otherItems = $otherItems;
+		$this->token = $token;
```

Comment:
(I’d prefer to move this up one line, so that the field, constructor params, assignments and getters are all in the same order.)

Reply:
done